docs/cloud/features/scheduler/scheduler.md

```diff
@@ -163,8 +163,35 @@ This gives you complete control over data security and network access while stil
 
 ### How it works
 
-Coming soon!
+Self-hosted executors as the name indicates are self-hosted workers that take on the responsibility of "executing" changes to the data warehouse. While Tobiko Cloud schedules and plans changes, the executors are responsible for executing those changes. Executors are docker containers that are configured to connect to Tobiko Cloud as well as your data warehouse. Connected to both, the executor will pull the work, whether it's a plan or scheduled background work from a run, and execute it on your data warehouse.
 
 ### Configuration
 
-Coming soon!
\ No newline at end of file
+Exact configuration is left to the user and will vary based on the infrastructure and setup of the user, for example the executors could be run on a Kubernetes cluster or as a standalone pair of Docker containers depending on the user's infrastructure.
+
+Tobiko Cloud requires 2 docker instances to be running, one for runs and one for plans. The entrypoint for both is `executor run` and `executor plan` respectively. The executor container can be found on [Docker Hub](https://hub.docker.com/r/tobikodata/tcloud). In addition to running the container, the user will need to configure the executor with environment variables that point to the cloud sqlmesh instance as well as the data warehouse.
+
+To connect to the cloud SQLMesh instance, the user will need to provide the following environment variables:
+
+```env
+TCLOUD_URL=https://cloud.tobikodata.com/sqlmesh/acme/analytics_project
+TCLOUD_TOKEN=your_token
+```
+
+In addition to the above variables, a gateway is needed to provide a connection to a data warehouse. The following details are an example of how this is done for a Postgres data warehouse:
```

Comment:
We need a link to connection configuration (https://sqlmesh.readthedocs.io/en/stable/integrations/overview/#execution-engines) as well as a link to the page which explains the env variable name format (https://sqlmesh.readthedocs.io/en/stable/guides/configuration/#overrides)

Reply:
Done 👍